.NET/Microsoft.Recognizers.Text.Sequence/Chinese/Extractors/ChineseQuotedTextExtractor.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Recognizers.Text.Sequence.Chinese
+{
+    public class ChineseQuotedTextExtractor : BaseQuotedTextExtractor
+    {
+        private static Dictionary<Regex, string> regexes = new Dictionary<Regex, string>
+            {
+                {
+                    new Regex(@"(“([^“”]+)”)"),
```

Comment:
These regex declarations should be in a YAML file for Chinese. Can you refactor these before adding new languages?
Should this with double-quotes be in Base though as it would fir multiple languages?